### PR TITLE
Link to the widget catalog from the dartdocs

### LINKED ISSUE #7988: Material library API docs doesn't link to our Widgets Overview page
This page: https://docs.flutter.io/flutter/material/material-library.html should probably link over to https://flutter.io/widgets/

The widgets overview page is a very helpful resource.

### PR DESCRIPTION
**Pull request by Hixie**
Fixes https://github.com/flutter/flutter/issues/7988

cc @sethladd

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

**Comment by sethladd** [comment]
Cool, thanks!

File: packages/flutter/lib/material.dart [hunk 1]
```diff
@@ -6,9 +6,12 @@
 ///
 /// To use, import `package:flutter/material.dart`.
 ///
-/// See
-/// [material.google.com](https://material.google.com/)
-/// for an introduction to Material Design.
+/// See also:
+///
+///  * [material.google.com](https://material.google.com/)
```
Comment (sethladd) [inline packages/flutter/lib/material.dart]: I'd recommend swapping the order of these links. The /widgets should be much more useful for a Flutter developer.

Comment (Hixie) [inline packages/flutter/lib/material.dart]: done

**Pull request merged**